Frontend/Camera2Basic/ErrorDialog.cs

```diff
@@ -0,0 +1,37 @@
+
+using Android.App;
+using Android.Content;
+using Android.OS;
+
+namespace Camera2Basic
+{
+    public class ErrorDialog : DialogFragment
+    {
+        private static readonly string ARG_MESSAGE = "message";
+        private static Activity mActivity;
+
+        private class PositiveListener : Java.Lang.Object, IDialogInterfaceOnClickListener
```

Comment:
You are using positive/negative listeners in multiple classes, what about creating an interface with method OnClick and using dependency injection there?